src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -200,28 +205,107 @@ private async Task<RoatpFinancialApplicationViewModel> CreateRoatpFinancialAppli
+
+            return activelyTradingSection;
+        }
+
+        private async Task<Section> GetOrganisationTypeSection(Guid applicationId)
+        {
+            const string OrganisationTypeSectionTitle = "Organisation type";
+            const string OrganisationTypeMainSupportingPageId = "140";
+            const string OrganisationTypeEmployerPageId = "150";
+
+            Section organisationTypeSection = await _qnaApiClient.GetSectionBySectionNo(applicationId, OrganisationDetailsSequenceNo, 4);
+            organisationTypeSection.LinkTitle = OrganisationTypeSectionTitle;
+            organisationTypeSection.Title = OrganisationTypeSectionTitle;
+            organisationTypeSection.QnAData.Pages = organisationTypeSection.QnAData.Pages?.Where(page => page.PageId == OrganisationTypeMainSupportingPageId || page.PageId == OrganisationTypeEmployerPageId).ToList();
+
+            return organisationTypeSection;
+        }
+
+        private async Task<List<Section>> GetFinancialSections(Guid applicationId)
+        {
+            const string CompanyFhaSectionTitle = "Organisation's financial health";
+            const string ParentCompanyFhaSectionTitle = "UK ultimate parent company's financial health";
+
+            var financialSections = new List<Section>();
+
             var roatpSequences = await _applyApiClient.GetRoatpSequences();
             var financialSequences = roatpSequences.Where(x => x.Roles.Contains(Roles.ProviderRiskAssuranceTeam));
-            var financialSections = new List<Section>();
-            foreach(var sequence in financialSequences)
+           
+            foreach (var sequence in financialSequences)
             {
-                var qnaSequence = await _qnaApiClient.GetSequenceBySequenceNo(applicationFromAssessor.ApplicationId, sequence.Id);
-                var sections = await _qnaApiClient.GetSections(applicationFromAssessor.ApplicationId, qnaSequence.Id);
-                foreach(var section in sections)
+                var qnaSequence = await _qnaApiClient.GetSequenceBySequenceNo(applicationId, sequence.Id);
+                var sections = await _qnaApiClient.GetSections(applicationId, qnaSequence.Id);
+                foreach (var section in sections)
                 {
                     var roatpSequence = roatpSequences.FirstOrDefault(x => x.Id == sequence.Id);
 
                     if (!roatpSequence.ExcludeSections.Contains(section.SectionNo.ToString()))
                     {
-                        financialSections.Add(section);
+                        // Ensure at least one active page is answered
+                        if (section.QnAData.Pages.Any(p => p.Active && p.Complete))
+                        {
+                            // APR-1477 - adjust title for Assessor
+                            if(section.SequenceNo == FinancialHealthSequenceNo && section.SectionNo == 2)
```

Comment:
Can we replace magic section numbers with constants, and have them all in a logical place?

Reply:
done